### PR TITLE
add download as csv with filter

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

File: mentorship/mentor/admin.py [hunk 1]
```diff
@@ -5,15 +5,88 @@
...
+            ('This Year', _('This Year')),
+        )
+
+    def queryset(self, request, queryset):
+        """
+        Returns the filtered queryset based on the value
+        provided in the query string and retrievable via
+        `self.value()`.
+        """
+        global DATE_RANGE
```
Comment (miltontony) [inline mentorship/mentor/admin.py]: Definitely -100 on this. Please find a way of doing this without `global`

Comment (codiebeulaine) [inline mentorship/mentor/admin.py]: @miltontony yeah, haven't implemented your suggestion. Will do so once it works

File: mentorship/mentor/admin.py [hunk 2]
```diff
@@ -5,15 +5,88 @@
...
+    else:
+        next_month = today.replace(month=today.month + 1, day=1)
+    next_year = today.replace(year=today.year + 1, month=1, day=1)
+
+    if DATE_RANGE == 'Any Date' or DATE_RANGE == '':
+        return queryset
+    if DATE_RANGE == 'Today':
+        return queryset.filter(**{field_name: today})
+    if DATE_RANGE == 'Past 7 Days':
+        return queryset.filter(field_name__gte=(today - datetime.timedelta(
```
Comment (codiebeulaine) [inline mentorship/mentor/admin.py]: @hodgestar @miltontony the problem I'm having is here, I'm not sure how to reference the dynamic field name as well as the gte filter

File: mentorship/mentor/admin.py [hunk 3]
```diff
@@ -5,15 +5,88 @@
 from django.db.models import Q
 from django.utils.html import format_html
 from django.core.urlresolvers import reverse
+import csv
```
Comment (hodgestar) [inline mentorship/mentor/admin.py]: Can we put all the standard library imports at the top?

**Comment by hodgestar** [comment]
Just noting that we closed this because it turns out the actual feature request is to export calls and messages (see #85).

**Pull request closed without merging**